src/js/lib/admin-page/style.scss

```diff
@@ -0,0 +1,165 @@
+$gray: #777;
+
+#wp-parsely-react-entrypoint {
+	.components-snackbar {
+		border-left: 5px solid green;
+		&.settings-error {
+			border-left: 5px solid red;
+		}
+	}
+	#wp-parsely_version {
+		color: $gray;
```

Comment:
Will this grey text work on the various different wp-admin colour schemes and give sufficient colour contrast?